### PR TITLE
Invoice item repo feature

### PR DESCRIPTION
**Pull request by klatour324**
Here is the Invoice Item and Invoice Item Repo classes. Take a look and check to make sure all looks good and our methods a good length of 7 or less lines.

### CODE REVIEW

File: lib/invoice_item_repo.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                          item_id:  attributes[:item_id],
+                          invoice_id:  attributes[:invoice_id],
+                          quantity: attributes[:quantity],
+                          unit_price:  attributes[:unit_price],
+                          created_at:  Time.now,
+                          updated_at:  Time.now
+                          }))
+  end
+
+  def update(id, attributes)
```
Comment (krislitman) [inline lib/invoice_item_repo.rb:54]: We should come back to these update methods and try to refactor, to get our method line count down!

**Comment by IcelandicGambit** [comment]
Methods look clean and terse.

**Comment by krislitman** [comment]
Everything looks good and fixed the merge conflict!

**Comment by rcasias** [comment]
Great work @klatour324 !

**Pull request merged**